app/assets/stylesheets/application.css.scss

```diff
@@ -203,15 +215,45 @@ main {
   box-shadow: 1px 1px 10px 1px rgba(0,0,0,0.7);
 }
 
-.town_table td{
+// .town_table{
+//   max-width: 800px !important;
+//   display: block;
+// }
+//
+// .town_table td{
+//   border: none;
+//   border-collapse: collapse;
+//   border-spacing: 0px;
+//   padding-bottom: 0px;
+// }
+//
+// .town_table img.responsive-img {
+//   max-width: inherit !important;
+//   // max-width: 90% !important;
+//   height: auto;
+//   width: auto;
+//   vertical-align: bottom;
+//   display: block;
+//   padding-right: 0px;
+//   margin-right: 0px;
+// }
+.town.row{
+  max-width: 800px !important;
```

Comment:
!important should not be used<br>Properties should be ordered border, border-collapse, border-spacing, display, margin-bottom, max-width, padding-bottom